### PR TITLE
[MRG] fix overlaping subplots in evoked histograms

### LINKED ISSUE #5234: Wrong title layout when calling historgram
When calling histogram by `h` in an epochs plot the axis labels do not render properly. 

```py
import mne
data_path = mne.datasets.sample.data_path()
raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
raw = mne.io.read_raw_fif(raw_fname)
events = mne.find_events(raw, stim_channel='STI 014')
picks = mne.pick_types(raw.info, meg=True, eeg=True, eog=True, stim=False, exclude='bads')
mag_picks = mne.pick_types(raw.info, meg='mag', eog=True, exclude='bads')
grad_picks = mne.pick_types(raw.info, meg='grad', eog=True, exclude='bads')
event_id = dict(aud_l=1, aud_r=2)  # event trigger and conditions
tmin = -0.2  # start of each epoch (200ms before the trigger)
tmax = 0.5  # end of each epoch (500ms after the trigger)event_id = dict(aud_l=1, aud_r=2)  # event trigger and conditions
reject = dict(grad=4000e-13, mag=4e-12, eog=150e-6)
epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True, picks=picks,preload=False, reject=reject)
epochs.plot()

```

image

### PR DESCRIPTION
**Pull request by massich**
fixes #5234 
image

### CODE REVIEW

File: mne/viz/epochs.py [hunk 1]
```diff
@@ -1967,7 +1967,7 @@ def _plot_histogram(params):
             ax.plot((rej, rej), (0, ax.get_ylim()[1]), color='r')
         plt.title(titles[types[idx]])
     params['histogram'].suptitle('Peak-to-peak histogram', y=0.99)
-    params['histogram'].subplots_adjust(hspace=0.6)
+    params['histogram'].subplots_adjust(hspace=1)
```
Comment (agramfort) [inline mne/viz/epochs.py]: why not a call to tight_layout?

Comment (massich) [reply mne/viz/epochs.py]: 'cos it messes up with the title anyway
image

**Comment by jona-sassenhagen** [comment]
How does it look for the default render to PNG in Jupyter notebooks?

**Comment by massich** [comment]
@jona-sassenhagen I don't use that much jupyter notebooks, how do you manage to trigger the histogram from here? 

image

**Comment by jona-sassenhagen** [comment]
Ah, this is for interactive stuff?

Try
`%matplotlib notebook`

**Comment by massich** [comment]
Nop, now I can interact with the mouse but I cannot input the keyboard (I've deactivated all vimium and so on, but no chance)

**Comment by massich** [comment]
@larsoner @agramfort 
This is how it looks like when `plt.tight_layout(h_pad=0, pad=2)`
image

**Comment by massich** [comment]
`h_pad=0.7` looks bit better. But at this point IMHO if some user does not like the default, they can change it themselves with the handle.
image

**Comment by agramfort** [comment]
Last figure looks good

**Comment by agramfort** [comment]
thx @massich

**Pull request merged**